parent/pom.xml

```diff
@@ -3546,12 +3546,12 @@
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <configuration>
                         <attach>true</attach>
-                        <source>${jdk.version}</source>
+                        <release>${maven.javadoc.release}</release>
```

Comment:
If I understood well, we should be able to even not provide the release version now as it would pick it up from maven-compiler.release https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#release
But maybe better to keep it for now to have a finer grained control.